apkid/rules/elf/packers.yara

```diff
@@ -289,3 +289,30 @@ rule upx_unknown_version_unmodified : packer
     upx_unmodified and
     not upx_compressed_apk
 }
+
+rule ollvm_obfuscator : packer embedded
+{
+  meta:
+    description = "Obfuscator-LLVM ELF embedded in APK"
```

Comment:
Could we get a URL and hash here also?

Reply:
I already added at https://github.com/rednaga/APKiD/issues/5 Is this enough?
The ollvm detection on shared objects has not been tested. We should review it together.